physical/spanner/spanner.go

```diff
@@ -143,14 +143,14 @@ func NewBackend(c map[string]string, logger log.Logger) (physical.Backend, error
 		return nil, errwrap.Wrapf("failed to parse max_parallel: {{err}}", err)
 	}
 
-	logger.Debug("configuration",
+	logger.Debug("physical/spanner: configuration",
```

Comment:
Please don't do this: we already pass in a named logger which should give a prefix of `storage.spanner` in front of each log line.

Reply:
Ah okay - then I updated this to remove the prefix from GCS @jefferai